templates/addons/aurora/mysql-cf.yml

```
@@ -66,77 +58,81 @@ Resources:
               StringEquals:
                 'kms:CallerAccount': !Ref 'AWS::AccountId'
                 'kms:ViaService': !Sub 'rds.${AWS::Region}.amazonaws.com'
-
   AuroraKMSCMKAlias:
     Type: 'AWS::KMS::Alias'
     DeletionPolicy: Retain
-    DependsOn: ['AuroraDBCluster']
+    DependsOn: ['{{logicalIDSafe .Name}}']
     Properties:
-      AliasName: !Sub 'alias/${AuroraDBCluster}'
+      AliasName: !Sub 'alias/${ {{- logicalIDSafe .Name -}} }'
       TargetKeyId: !Ref AuroraKMSCMK
-
   DBSubnetGroup:
     Type: 'AWS::RDS::DBSubnetGroup'
     Properties:
-      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster. 
+      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster.
       SubnetIds:
         !Split [',', { 'Fn::ImportValue': !Sub '${App}-${Env}-PrivateSubnets' }]
-
   DBClusterSecurityGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: The Security Group for the database cluster. 
+      GroupDescription: The Security Group for the database cluster.
       SecurityGroupIngress:
-        - ToPort: !Ref DBPort
-          FromPort: !Ref DBPort
+        - ToPort: 3306
+          FromPort: 3306
           IpProtocol: tcp
           Description: From Environment Security Group
-          SourceSecurityGroupId: 
-            Fn::ImportValue: 
+          SourceSecurityGroupId:
+            Fn::ImportValue:
               !Sub '${App}-${Env}-EnvironmentSecurityGroup'
-      VpcId: 
-        Fn::ImportValue: 
+      VpcId:
+        Fn::ImportValue:
           !Sub '${App}-${Env}-VpcId'
       SecurityGroupEgress:
         - IpProtocol: '-1'
           Description: Allow all outbound traffic by default
           CidrIp: 0.0.0.0/0
-
   AuroraSecret:
     Type: AWS::SecretsManager::Secret
     Properties:
-      Name: !Sub '/${App}/${Env}/${Name}/${DBTableName}'
-      Description: !Sub Aurora PostgreSQL main user secret for ${AWS::StackName}
+      Name: !Sub '/${App}/${Env}/${Name}'
```

We may need to add the tablename to this secret to avoid the chance of collisions. Or perhaps to just do
```suggestion
      Name: !Sub '/${App}/${Env}/${Name}/MySQLSecret'
```